### PR TITLE
feat: add adr content to specify the Map in any realms

### PR DESCRIPTION
**Pull request by leanmendoza**
Following spec
https://github.com/decentraland/adr/pull/266

### CODE REVIEW

File: proto/decentraland/realm/about.proto [hunk 1]
```diff
@@ -11,9 +13,57 @@ message AboutResponse {
...
-    optional string data_image = 2;
-    optional string estate_image = 3;
+
+    // deprecated parameters
+    reserved 2;
+    reserved "data_image";
+    reserved 3;
+    reserved "estate_image";
+  
+    // the union of all rects here represents the places where can have scenes
```
Comment (marianogoldman) [inline proto/decentraland/realm/about.proto:25]:
```suggestion
    // the union of all rects here represents the places that can have scenes
```

File: proto/decentraland/realm/about.proto [hunk 2]
```diff
@@ -11,9 +13,57 @@ message AboutResponse {
...
+    //  - all the other parcels are considered as empty, up to the explorer to decide if they're walkable or not
+    //  - the contained square is determined by points: 
+    //    - top-left      with  minX, maxY 
+    //    - bottom-left   with  minX, minX
```
Comment (marianogoldman) [inline proto/decentraland/realm/about.proto:29]:
```suggestion
    //    - bottom-left   with  minX, minY 
```

File: proto/decentraland/realm/about.proto [hunk 3]
```diff
@@ -11,9 +13,57 @@ message AboutResponse {
...
+    reserved 3;
+    reserved "estate_image";
+  
+    // the union of all rects here represents the places where can have scenes
+    //  - all the other parcels are considered as empty, up to the explorer to decide if they're walkable or not
+    //  - the contained square is determined by points: 
+    //    - top-left      with  minX, maxY 
+    //    - bottom-left   with  minX, minX 
+    //    - top-right     with  maxX, maxY 
+    //    - bottom-right  with  maxX, maxY
```
Comment (marianogoldman) [inline proto/decentraland/realm/about.proto:31]:
```suggestion
    //    - bottom-right  with  maxX, minY 
```

**Review by kuruk-mm — DISMISSED** [review]
Looks great to me! 

Just apply @marianogoldman review

File: proto/decentraland/realm/about.proto [hunk 4]
```diff
@@ -10,10 +13,55 @@ message AboutResponse {
...
-    optional string estate_image = 3;
+  message MapConfiguration {
+    // wheter the minimap could be rendered or not
+    bool minimap_enabled = 1;
+  
+    // the union of all rects here represents the places where that MAY have scenes
+    //  - all the other parcels are considered as empty, up to the explorer to decide if they're walkable or not
+    //  - the contained square is determined by points: 
+    //    - top-left      with  minX, maxY 
+    //    - bottom-left   with  minX, minX
```
Comment (kuruk-mm) [inline proto/decentraland/realm/about.proto:24]:
```suggestion
    //    - bottom-left   with  minX, minY
```

I think the review was not applied correctly

File: proto/decentraland/realm/about.proto [hunk 5]
```diff
@@ -10,10 +13,55 @@ message AboutResponse {
...
+    // wheter the minimap could be rendered or not
+    bool minimap_enabled = 1;
+  
+    // the union of all rects here represents the places where that MAY have scenes
+    //  - all the other parcels are considered as empty, up to the explorer to decide if they're walkable or not
+    //  - the contained square is determined by points: 
+    //    - top-left      with  minX, maxY 
+    //    - bottom-left   with  minX, minX 
+    //    - top-right     with  maxX, maxY 
+    //    - bottom-right  with  maxX, maxY
```
Comment (kuruk-mm) [inline proto/decentraland/realm/about.proto:26]:
```suggestion
    //    - bottom-right  with  maxX, minY 
```

same

**Review by kuruk-mm — DISMISSED** [review]

File: proto/decentraland/realm/about.proto (hunk 4 above)
Comment (leanmendoza) [reply proto/decentraland/realm/about.proto]: thanks

**Review by robtfm — APPROVED** [review]
lgtm

for genesis.city the max zoom level is 5, but i don't think it needs to be config (and explorers should fall back to lower levels if the higher ones aren't found anyway). might be worth a comment though?

**Review by kuruk-mm — APPROVED** [review]
Great!

**Review by pravusjif — APPROVED** [review]
LGTM, nice job!

**Pull request merged**